nautobot_golden_config/nornir_plays/config_deployment.py

```diff
@@ -34,7 +37,16 @@ def run_deployment(task: Task, logger: NornirLogger, commit: bool, config_plan_q
             config=consolidated_config_set,
             default_drivers_mapping=get_dispatcher(),
         )[1].result["result"]
-        logger.log_success(obj=obj, message="Successfully deployed configuration to device.")
+        if not result:
+            plans_to_deploy.update(status=Status.objects.get(slug="failed"))
+            logger.log_failure(obj=obj, message="No Nornir Result was Returned.")
+        else:
+            if not result.failed:
+                logger.log_success(obj=obj, message="Successfully deployed configuration to device.")
+                plans_to_deploy.update(status=Status.objects.get(slug="completed"))
+            else:
+                plans_to_deploy.update(status=Status.objects.get(slug="failed"))
+                logger.log_failure(obj=obj, message="Failed deployment to the device.")
```

Comment:
Are we sure this is working correctly? I ran a job using the sandbox data (which will never connect) and I got the following error:
> Failed with an unknown issue. Cannot connect to jcy-spine-02.infra.ntc.com

The Config Plans that I tried to deploy were still in the `In Progress` status